scripts/generate_json_docs.py

```diff
@@ -107,7 +103,7 @@ def to_dict(self):
                 'description': format_sphinx_doc(self.description),
                 'examples': self.examples,
                 'methods': [m.to_dict() for m in self.methods],
-                'source': self.source}
+                'source': '%s' % self.source}
```

Comment:
Ah sorry this is already a string. I had an error I was debugging and I forgot to put this back.